app/uk/gov/hmrc/residencenilratebandcalculator/views/rnrbHelpers/input_date.scala.html

```diff
@@ -1,12 +1,14 @@
-@(args: (Symbol, Any)*)(implicit messages: Messages)
-@inputArgs = @{ args.toMap }
-@id = @{ inputArgs.getOrElse('_id, "").toString }
-@label = @{ inputArgs.getOrElse('_label, "").toString }
-@valueDay = @{ inputArgs.getOrElse('_valueDay, "").toString }
-@valueMonth = @{ inputArgs.getOrElse('_valueMonth, "").toString }
-@valueYear = @{ inputArgs.getOrElse('_valueYear, "").toString }
-@hint = @{ inputArgs.getOrElse('_hint, "").toString }
-@errorKey = @{ inputArgs.getOrElse('_error, "").toString }
+@(
+label: String,
+errorKey: String,
+value: Option[String] = None,
```

Comment:
Is the value parameter needed?  I imagine we'll always use the separate day/month/year parameters instead?